src/oracle/query_dsl/returning.rs

```diff
@@ -92,6 +92,8 @@ macro_rules!  impl_bind_column_list {
                         }
                         out.push_sql(":");
                         out.push_sql($T::NAME);
+                        // Fixes ORA-01745, if T::NAME is a reserved word.
```

Comment:
Maybe we should just use normal numbered bind's here?

Reply:
I tried the following which gave me this error: `attempted to repeat an expression containing no syntax variables matched as repeating at this depth`. Didn't find out what is the problem

```rust
let mut needs_comma = false;
let mut count = 1;
$(
	if needs_comma {
		 out.push_sql(", ");
	}
	let placeholder = format!(":out{}", count);
	out.push_sql(&placeholder);
	needs_comma = true;
	count += 1;
)+
```
Not quite sure what this means.

Reply:
I tried the following which gave me this error: `attempted to repeat an expression containing no syntax variables matched as repeating at this depth`. Didn't find out what is the problem

```rust
let mut needs_comma = false;
let mut count = 1;
$(
	if needs_comma {
		 out.push_sql(", ");
	}
	let placeholder = format!(":out{}", count);
	out.push_sql(&placeholder);
	needs_comma = true;
	count += 1;
)+
```
Not quite sure what this means.

Reply:
@geomaniac was this an Oracle or rustc error?

Reply:
rustc

Reply:
This means that the inner expression needs to at least one of the following macro variables: `$idx`, `$T`, `$ST` or `$TT`. We want to use `$idx` in this case because it contains the number needed for the format.
This also allows us to remove the count variable :wink:

Reply:
ah ok thanks, I'll try if this work. Rust macros are an area I have yet to explore.